helm/alfresco-content-services/values.yaml

```diff
@@ -549,29 +548,33 @@ alfresco-ai-transformer:
 elasticsearch:
   # -- Enables the embedded elasticsearch cluster
   enabled: true
-  replicas: 1
-  clusterHealthCheckParams: "wait_for_status=yellow&timeout=1s"
-elasticsearch-audit:
-  # -- Enables the embedded elasticsearch cluster for alfresco-audit-storage
-  enabled: true
-  nameOverride: elasticsearch-aas
-  replicas: 1
-  clusterHealthCheckParams: "wait_for_status=yellow&timeout=1s"
-  clusterName: elasticsearch-aas
-  extraEnvs:
-    - name: ELASTIC_USERNAME
-      valueFrom:
-        secretKeyRef:
-          name: *aas_elasticsearch_secretName
-          key: AUDIT_ELASTICSEARCH_USERNAME
-    - name: ELASTIC_PASSWORD
-      valueFrom:
-        secretKeyRef:
-          name: *aas_elasticsearch_secretName
-          key: AUDIT_ELASTICSEARCH_PASSWORD
+  image:
+    tag: 7.17.26
+  master:
+    masterOnly: false
+    replicaCount: 1
+  data:
+    replicaCount: 0
+  coordinating:
+    replicaCount: 0
+  ingest:
+    replicaCount: 0
+  kibana:
+    image:
+      tag: 7.17.26
+    configuration:
+      server:
+        basePath: "/kibana"
+        publicBaseUrl: "http://localhost/kibana"
```

Comment:
That looks a bit odd to me. Did you check wether kibana was properly accessible in integration pipeline?